### PR TITLE
chore: Make the server a standalone project

### PR DESCRIPTION
**Pull request by atomrc**

### CODE REVIEW

File: grunt/config/compress.js [hunk 1]
```diff
@@ -22,14 +22,16 @@
...
+  server: {
+    files: [
+      {
+        cwd: '<%= dir.dist_ %>',
+        expand: true,
+        src: ['../package.json', '**/*', '.**/*', '!<%= dir.dist.s3 %>'],
+      },
+    ],
+    options: {
+      archive: '<%= dir.dist.s3 %>/ebs.zip',
```
Comment (Yserz) [inline grunt/config/compress.js:34]: I assume we have to change the `travis.yml` for this change

File: dist/apple-app-site-association.json [hunk 2]
```diff
@@ -1,11 +0,0 @@
-{
```
Comment (Yserz) [inline dist/apple-app-site-association.json]: Let's double check that this file is still existent in the `server/dist` folder

File: package.json [hunk 3]
```diff
@@ -1,20 +1,7 @@
...
-    "express-useragent": "1.0.12",
-    "fs-extra": "7.0.1",
-    "geolite2": "1.3.0",
-    "hbs": "4.0.1",
-    "helmet": "3.15.0",
-    "logdown": "3.2.7",
-    "maxmind": "2.7.0",
-    "pm2": "3.2.2"
-  },
+  "dependencies": {},
```
Comment (Yserz) [inline package.json:4]: With the server cleanly separated we could move all the dependencies to the correct place

File: server/.gitignore [hunk 4]
```diff
@@ -0,0 +1,6 @@
+node_modules/
+dist/*
+!dist/templates/
+!dist/robots/
+!dist/app-app-site-association.json
```
Comment (Yserz) [inline server/.gitignore]:
```suggestion
!dist/apple-app-site-association.json
```

File: server/package.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "wire-webapp-server",
```
Comment (Yserz) [inline server/package.json]: Lets call it `wire-web-server` because it will be used for other projects as well

File: package.json (hunk 3 above)
Comment (atomrc) [reply package.json:4]: Will do that in the base branch (chore/module_system)

File: package.json [hunk 6]
```diff
@@ -214,10 +199,10 @@
     "lint:js": "eslint -c .eslintrc.json --ignore-path .gitignore --ignore-path .eslintignore --ext .js app/script/ test/ bin/ grunt/",
     "lint:ts:auth": "tslint --config tslint.json --project tsconfig.json \"app/script/**/*.{ts,tsx}\"",
     "lint:ts:server": "tslint --config tslint.json --project tsconfig.server.json \"server/**/*.{ts,tsx}\"",
-    "postinstall": "node ./bin/postinstall.js",
```
Comment (Yserz) [inline package.json:217]: File can be removed

File: server/package.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "wire-web-server",
+  "version": "1.0.0",
+  "main": "index.js",
```
Comment (Yserz) [inline server/package.json]:
```suggestion
  "main": "dist/index.js",
```

File: server/package.json [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+  "license": "MIT",
```
Comment (Yserz) [inline server/package.json]: GPL-3.0?

**Pull request merged**